doc/project/tsc.rst

```diff
@@ -0,0 +1,218 @@
+- **Documentation:** Contribute to the development and maintenance of project
+  documentation, including technical guides, API references, and best practices.
+- **Release Management:** Collaborate with the release manager and other
+  stakeholders to plan and coordinate project releases, ensuring timely delivery
+  and quality assurance.
+- **Contributor Support:** Support and mentor new contributors, helping them
+  navigate the project's codebase, processes, and community norms.
+- **Issue Triage:** Assist in triaging and prioritizing issues reported by users and
+  contributors, facilitating timely resolution and communication.
+- **Compliance and Licensing:** Ensure compliance with project licensing
+  requirements and open-source best practices, addressing any licensing-related
+  issues that may arise.
+- **Conflict Resolution:** Facilitate constructive discussions and resolution of
+  technical disagreements or conflicts within the community, promoting a healthy
+  and inclusive environment.
+- **Continuous Improvement:** Continuously seek opportunities to improve project
+  governance, processes, and infrastructure, driving innovation and
+  sustainability.
+
+
+Appointed TSC Members
++++++++++++++++++++++
+
+See `Zephyr project charter`_ for more details.
+
+- Appointed TSC members have no term limits besides the term of their employment
+  at the organization they represent or their organization's membership in the
+  Zephyr Project.
+- Appointed TSC members can select an Alternate from the same organization.
+
+Elected TSC Members
++++++++++++++++++++
+
+Per the `Zephyr project charter`_, TSC members can nominate representatives from
+the technical community at the rate of no more than one per quarter.
+
+- Majority vote is required to confirm a candidate.
+- Once elected, a TSC member serves for 2 years.
+- Elected TSC members do not have the right to appoint an Alternate.
+- To ensure continuity of the TSC, at the end of the 2 year term, the TSC is
+  required to reconfirm the membership of elected members. If the elected member
+  declines a new term or if the TSC fails to reconfirm the term, the seat will
+  be open for new nominations.
+- If an elected TSC member resigns before the end of the 2 year term, their spot
+  will be open for new members outside of the quarterly nomination limit. The
+  elected member will serve a 2 year term.
+- The TSC has the right to terminate elected members who become inactive and are
+  not fulfilling the responsibilities of TSC members as described in this
+  document.
+- The number of elected members shall not exceed 20% of the total of appointed
+  members.
+- Existing TSC members who were elected before May 2024 shall be re-confirmed
+  after completing the 2 year term since they were first elected.
+
+Suspensions
++++++++++++
+
+As noted under Section 8b of the Project Charter, Voting rights for a
+representative who misses three consecutive meetings are subject to suspension
```

Comment:
From what I understood, rule applies to the representative and his alternate (meaning that both should miss 3 consecutive meetings for the voting right to be suspended).
Should this be clarified here or maybe in a dedicated section to clarify the role of the alternate in all these rights and duties?